tests/rptest/utils/rpenv.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+# Utilities for checking the environment of a test
+
+import os
+
+
+def sample_license():
+    """
+    Returns the sample license from the env if it exists, asserts if its
+    missing and the environment is CI
+    """
+    license = os.environ.get("REDPANDA_SAMPLE_LICENSE", None)
```

Comment:
i know this is only rearranging code so nothing really needs to change, but isn't the use of an environment variable to pass state into ducktape tests inconvenient? for example, locally, i'd have to restart my docker containers to make this work?

Reply:
Not sure, locally I've observed no issues, I run `REDPANDA_SAMPLE_LICENSE=xxx task rp:run-ducktape-tests ...` and it just works. Possibly because I've edited the taskfiles to pass the env var between environments?

In either case the rational for this was to be able to write tests against a 100yr valid license and not publicly expose it.

Reply:
i see. it works because the ducktape runner container is recreated every time.